src/test/java/tools/jackson/databind/convert/CoerceToBooleanTest.java

```diff
@@ -52,7 +52,9 @@ public BooleanWrapper(@JsonProperty("ctor") Boolean foo) {
         public void setPrimitive(boolean v) { primitive = v; }
     }
 
-    private final ObjectMapper DEFAULT_MAPPER = newJsonMapper();
+    private final ObjectMapper DEFAULT_MAPPER = jsonMapperBuilder()
```

Comment:
This test failure for `byte[]` looks suspicious; I am not sure it should fail. Others in this package were legit, merged change in 2.19 to ensure consistent test code.